gnosis/xml/xmlmap.py

```diff
@@ -26,13 +26,6 @@
 
 import re
 
-# define True/False if this Python doesn't have them (only
```

Comment:
😕 🤷

Reply:
I'm honestly wondering what version  of Python this was written for where booleans may or may not have existed 😂